engine/src/main/java/com/datatorrent/stram/engine/StreamingContainer.java

```diff
@@ -849,8 +849,8 @@ private void stopInputNodes()
         if (thread == null || !thread.isAlive()) {
           continue;
         }
+        node.shutdown(true);
```

Comment:
Oops, I was testing different PR on master before started working on it. Reverted these changes now.